engine/runtime/src/main/java/org/enso/interpreter/runtime/EnsoContext.java

```diff
@@ -478,9 +480,12 @@ public Object lookupJavaClass(String className) {
           return lookupHostSymbol(pkgName, fullInnerClassName);
         }
       } catch (RuntimeException | InteropException ex) {
-        logger.log(Level.WARNING, null, ex);
+        collectedExceptions.add(ex);
       }
     }
+    for (var ex : collectedExceptions) {
+      logger.log(Level.WARNING, null, ex);
```

Comment:
Print out all collected exceptions, return `null` and the system to throw Enso `PanicException`